### PR TITLE
CY-3549 Add 'mapping' option to plugin update

### PR DESCRIPTION
**Pull request by mateumann**

### CODE REVIEW

File: rest-service/dev-requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-git+https://github.com/cloudify-cosmo/cloudify-common@master#egg=cloudify-common[dispatcher]==master
+git+https://github.com/cloudify-cosmo/cloudify-common@master#egg=cloudify-common[dispatcher]==CY-3549-cfy-plugin-update-mapping
```
Comment (tehasdf) [inline rest-service/dev-requirements.txt:1]: remember

File: rest-service/manager_rest/plugins_update/manager.py [hunk 2]
```diff
@@ -209,36 +219,42 @@ def get_reevaluated_plan(self, blueprint, resolver_parameters):
         return temp_plan
 
 
+def _plugins_in_a_plan(plan):
+    for executor in [DEPLOYMENT_PLUGINS_TO_INSTALL,
```
Comment (tehasdf) [inline rest-service/manager_rest/plugins_update/manager.py:223]: plz choose a different name, `executor` already means "host_agent" or "central_deployment_agent", this name just makes it confusing

File: tests/dev-requirements.txt [hunk 3]
```diff
@@ -1,5 +1,5 @@
-https://github.com/cloudify-cosmo/cloudify-common/archive/master.zip
-https://github.com/cloudify-cosmo/cloudify-cli/archive/master.zip
+https://github.com/cloudify-cosmo/cloudify-common/archive/CY-3549-cfy-plugin-update-mapping.zip
```
Comment (tehasdf) [inline tests/dev-requirements.txt:1]: remember

File: rest-service/manager_rest/plugins_update/manager.py (hunk 2 above)
Comment (mateumann) [reply rest-service/manager_rest/plugins_update/manager.py:223]: sure (not that I invented that `executor` ;) )

File: tests/dev-requirements.txt (hunk 3 above)
Comment (mateumann) [reply tests/dev-requirements.txt]: done

File: rest-service/dev-requirements.txt (hunk 1 above)
Comment (mateumann) [reply rest-service/dev-requirements.txt]: done

**Review by tehasdf — APPROVED** [review]

**Pull request merged**